spec/controllers/catalog_controller_spec.rb

```diff
@@ -54,6 +54,53 @@
       end
     end
   end
+  describe 'POST submit_cart_updates_form on item' do
+    before(:each) do
+      @equipment_model = FactoryGirl.create(:equipment_model)
+      put :add_to_cart, id: @equipment_model.id
```

Comment:
The test is designed to check that the form is able to change item quantities in the cart, so it needs to add an item into the cart first.